compose/ci.yaml

```diff
@@ -6,4 +6,4 @@ services:
     image: fuzzbench
     links:
       - queue-server
-    command: python3 -m pytest -vv -k test_e2e_run
+    command: python3 -m pytest -vv fuzzbench/test_e2e_run.py
```

Comment:
Did the `-k` format not work?

Reply:
It did not. (ran all the other tests in the new files that the pr adds)